data/database.py

```diff
@@ -2035,6 +2036,19 @@ class RepositoryAutoPrunePolicy(BaseModel):
     policy = JSONField(null=False, default={})
 
 
+class OauthAssignedToken(BaseModel):
+    uuid = CharField(default=uuid_generator, max_length=36, index=True, unique=True, null=False)
+    assigned_user = QuayUserField(index=True, unique=False, null=False)
+    application = ForeignKeyField(OAuthApplication, index=True, null=False, unique=False)
+    redirect_uri = CharField(max_length=255, null=True)
```

Comment:
does the redirect URI in this case contain the access token? if so consider encrypting this field

Reply:
The redirect URI should never contain any access token. The UI will set this to the config value `LOCAL_OAUTH_HANDLER` when it is being called [here](https://github.com/quay/quay/pull/2869/files#diff-2613f878141b14e270c68d1698010d584081398652c15f757e0dff579bf58617R104). If we feel that users may accidentally send tokens if they want to call this API themselves then i can add a check if need be